src/components/header.js

```diff
@@ -2,16 +2,26 @@ import './header.css';
 import { Link } from 'react-router-dom';
 
 const Header = () => (
-  <nav className="panel-bg">
-    <span className="nav-logo">Bookstore CMS</span>
-    <ul>
-      <li>
-        <Link to="/" className="nav-books"> Books</Link>
-      </li>
-      <li>
-        <Link to="/catagory" className="nav-category">Category</Link>
-      </li>
-    </ul>
-  </nav>
+  <div className="header">
+    <nav className="panel-bg">
+      <span className="nav-logo">Bookstore CMS</span>
+      <ul>
+        <li>
+          <Link to="/" className="nav-books">
+            {' '}
+            Books
+          </Link>
+        </li>
+        <li>
+          <Link to="/catagory" className="nav-category">
+            Category
```

Comment:
[REQUIRED] Kindly make this lighter and remove the underline just as it is in the template.